### ISSUE TITLE
Add notice to LERC T&Cs for LERC ES download filters

### ISSUE THREAD

**Issue by burkmarr** [BiologicalRecordsCentre/iRecord#931]
This new issue is branched from https://github.com/BiologicalRecordsCentre/iRecord/issues/800#issuecomment-694936703. We need a way to show a link to LERC T&Cs when a user selects a LERC filter from the ES [permissionFilters] control. @kitenetter has suggested identifying permission filters for LERCs by prefixing the title with 'LERC download - ', e.g:

- LERC download - BMERC
- LERC download - TVERC

**Comment by burkmarr** [comment]
@kitenetter - this is coded and available on iRecord dev. Rather than hard-code anything, I've added a new `@notices` attrbiute to the `[permissionFilters]` control. I've implemented an example on the demo page: http://test.brc.ac.uk/irecord_dev/es-download-dev.

If you select a filter in that control that starts with the word 'Verification' you will see a notice with a link displayed below the control. If you look at the configuration, you will see something like this:
```
@notices=<!--{
  "Verification": "<p><b>For LERC downloads, you must abide by the <a href='https://www.brc.ac.uk/irecord/lrc-download-2'>LERC Terms and Conditions</a>.</b></p>"
}-->
```
When this is implemented on the live site and you've created (or renamed) relevant filters so that they start with the text 'LERC download -', then we can add something like the above with the word 'Verification' replaced with 'LERC download -'. (The page being linked to at the moment is the old download - obviously you need something with just the T&C text.)

**Comment by kitenetter** [comment]
Related to this, can I pick up on a question to @johnvanbreda, who said that "using assignable download filters would be preferable" to our current system for providing LERC download access.

I propose to carry out the following to implement this - does this seem sensible:
1. Create a new role "LERC download" that has the same permissions as given to the "survey downloader" role.
2. For each LERC account that currently has "Regional collator" role, switch that to an "LERC download" role
3. Set up a appropriate "dataflow" filters, using the "LERC download" naming convention as described above, linked to each LERC account that needs a download.

**Comment by johnvanbreda** [comment]
Seems reasonable to me @kitenetter.

**Comment by kitenetter** [comment]
@burkmarr I have created a stand-alone page for the LERC download terms here:
https://www.brc.ac.uk/irecord/LERC-terms

The older version is still available here but I will remove this from the menu structure once the new system is in place:
https://www.brc.ac.uk/irecord/lrc-download-2

**Comment by burkmarr** [comment]
@kitenetter - when you create the LERC filters, will they be of type 'Data flow'?

**Comment by kitenetter** [comment]
@burkmarr yes they will

**Comment by kitenetter** [comment]
@burkmarr I've just set up a new filter "LERC download - Burkmar test" for your gmail account. It should contain spiders in Shropshire.

**Comment by burkmarr** [comment]
@kitenetter - thanks, that's working. I had to clear the iRecord Indicia cache before the new filter appeared in the permissionsFilter drop-down list - so just be aware of that.

**Comment by kitenetter** [comment]
Earlier in this thread I proposed these actions:

1. Create a new role "LERC download" that has the same permissions as given to the "survey downloader" role.
2. For each LERC account that currently has "Regional collator" role, switch that to an "LERC download" role
3. Set up appropriate "dataflow" filters, using the "LERC download" naming convention as described above, linked to each LERC account that needs a download.

Having experimented with things today, it appears that dataflow filters become available as a download option on the ES download page simply as a result of the filter existing, and no permissions are needed.

I **_think_** that is okay isn't it - we will only be setting up filters for the named users that we are giving the dataflow role to, and therefore it is fine if the filters, and not the role, controls what they can do.

Would be grateful for confirmation that I only need to create LERC filters, and not roles, and that the "survey downloader" role is now redundant. @johnvanbreda @burkmarr

**Comment by burkmarr** [comment]
Yes I think that's right Martin. As I understand it, the 'Records in collated locality download permission' method is superceded by permission filters, although it is still supported.

**Comment by johnvanbreda** [comment]
Correct. I would envisage this role not being ported to D8/9 at all.

**Comment by kitenetter** [comment]
Thanks. The first few LERC filters have been created and I await feedback from the LERC users! Closing.
